### ISSUE TITLE
if you add quality to item, search fails

### ISSUE THREAD

**Issue by nandotech** [SnosMe/awakened-poe-trade#10]
It appears any time I have an item with quality manually added to it (or from drops maybe, not sure as some were bought) prediction fails to pull up a price.

I assume this is an issue with the quality filter on api, but for example:

If I search a drop Corrupted Vaal Skeletons with 9% quality -- search works.

If I search a Level 1 23% Charged Traps search fails with the message "<item> is recognized as a valid poe item, but not supported currently!" Most skill gems seem to give this error. Not sure the cause/trigger here.

If I search a map with no quality (rare,regular,etc) it works fine. But if I search a map with chisels on it, I get the message again (see screenshot provided). This behavior is consistent for regular maps, blighted maps. 

![PathOfExile_x64Steam_aZLq41sMkb](https://user-images.githubusercontent.com/2347606/72122894-88ce7c80-332d-11ea-9515-ae4be399d80b.jpg)

Now a few pieces of armor have a separate issue, I can not put my finger on the cause of this one and the error message reads: "ML Prediction Error. Report prices to poeprices forum thread."

![PathOfExile_x64Steam_alr13JcuTN](https://user-images.githubusercontent.com/2347606/72122903-8c620380-332d-11ea-8bbe-3a294266fd53.jpg)

**Comment by SnosMe** [comment]
Yes, the quality on the item leads to an error, I knew about it. Sorry for not fixing it yet. This is due to the word "Superior" in the name.

As for the gem level 1 quality + 23%, this has already been fixed. It looks like you have an old version of the program. Please restart it once a day so that it can install the update (it is checked only when the program starts).

> ML Prediction Error
Everything is as it is. This is a response from poeprices.info. Perhaps their server was overloaded. The probability that I gave them the wrong text of the item is small. But still, I’ll ask you to write the Item info (Ctrl + C) of your helmet here.

**Comment by nandotech** [comment]
First let me say absolutely great job with all of this, I absolutely love the tool. I wouldn't even use poe trade macro because I hated it.

It is 100% reproducible. Just got it to happen again. Pasting item below. 

Also I just restarted the app (watched it load leagues etc) but most skill gems are still not being recognized if they are not level 20 appears to be the thing. Level 17 20 qualiy, level 1 20 quality. Even level 1 0 quality, same error (Recognized as value PoE item, but not supported.)

The helmet in question (and there are a few random other items I can provide, though as you mentioned it is likely the API response maybe due to so many mods):

Rarity: Rare
Morbid Brow
Bone Helmet
--------
Quality: +20% (augmented)
Armour: 236 (augmented)
Energy Shield: 74 (augmented)
--------
Requirements:
Level: 76
Str: 76
Int: 159
--------
Sockets: B-B-B-B 
--------
Item Level: 86
--------
Minions deal 20% increased Damage (implicit)
--------
+3 to Level of Socketed Minion Gems
Socketed Gems are Supported by Level 18 Minion Damage
+24 to maximum Energy Shield
+30 to maximum Life
Regenerate 4.6 Life per second
26% increased Stun and Block Recovery
Minions deal 19% increased Damage
+15% to Fire and Chaos Resistances (crafted)
--------
Shaper Item
Elder Item
